azure-spring-boot-bom/pom.xml

```diff
@@ -61,42 +61,42 @@
             <dependency>
                 <groupId>com.microsoft.azure</groupId>
                 <artifactId>azure-spring-boot-starter</artifactId>
-                <version>0.1.9-SNAPSHOT</version>
+                <version>${project.version}</version>
```

Comment:
I guess this won't work. ${project.version} will be evaluated to the version of user's module.

Reply:
You can take a try.

Reply:
Please don't use `${project.version}` in a BOM that is sending the wrong message. It's much more obvious to have the actual version.